etc/cron.d/bedrock-stage.tmpl

```diff
@@ -26,3 +26,5 @@ MAILTO="webops-cron@mozilla.com,cron-bedrock@mozilla.com"
 # bug 1087533
 42 * * * * {{ django_cron }} update_reps_ical > /dev/null 2>&1
 
+# bug 1128587
+38 * * * * {{ django_manage }} runscript update_firefox_os_feeds > /dev/null 2>&1
```

Comment:
Filed [bug 1133917](https://bugzilla.mozilla.org/show_bug.cgi?id=1133917) to fix the gen-crons.py script.